### PR TITLE
Make running against the emulator the default for Firestore integration tests

### PR DESCRIPTION
**Pull request by thebrianchen**
Currently `yarn test:emulator` downloads and runs the emulator, which is necessary when CI runs tests. However, there is currently no way to run tests against a locally running emulator. 

This adds the option to run tests against a locally running emulator on port 8080, which matches the Android SDK's functionality.

### CODE REVIEW

File: packages/firestore/CONTRIBUTING.md [hunk 1]
```diff
@@ -16,10 +18,30 @@ See
...
+    ```
+    firebase setup:emulators:firestore
+    ```
+  * Run the emulator
+    ```
+    firebase serve --only firestore
+    ```
 
-## Running Firestore Tests
+### Running Integration Tests
```
Comment (mikelehen) [inline packages/firestore/CONTRIBUTING.md]: We cover both unit and integration tests below so not sure "Integration" is appropriate.

File: packages/firestore/CONTRIBUTING.md [hunk 2]
```diff
@@ -43,7 +65,17 @@ yarn test:browser --integration
...
 # Run a subset of tests whose names match a filter.
 yarn test:browser --grep 'SortedSet keeps elements in the right order'
 yarn test:node --grep 'SortedSet keeps elements in the right order'
+
+# Run tests against against a Firestore emulator (emulator will be downloaded).
+yarn test:emulator
+
+# Run tests against a Firestore emulator running on localhost:8080.
+yarn test:node:emulator
+yarn test:node:persistence:emulator
```
Comment (mikelehen) [inline packages/firestore/CONTRIBUTING.md]: It seems weird that test:emulator and test:node:emulator behave differently wrt the emulator.  I think "node" and "emulator" are separate concerns and adding ":node" shouldn't affect the behavior of ":emulator".

So I think this argues for having ":emulator" and ":running-emulator" or something...  (or ":downloaded-emulator" and ":emulator")

**Comment by mikelehen** [comment]
FWIW another alternative to ":running-emulator" is renaming "test:emulator" to "test:travis" or something, if travis is the only place we want to download the emulator manually.  But again, I'm not clear on the intention.

**Comment by thebrianchen** [comment]
After an offline conversation with @mikelehen, we decided to make running tests against the emulator the default, which is the case for the Android and iOS SDKs. `yarn test:emulator` is renamed to `yarn test:travis` to avoid confusion between downloaded vs. running emulators.

File: packages/firestore/CONTRIBUTING.md [hunk 3]
```diff
@@ -16,10 +18,28 @@ See
 [Automated Setup](https://github.com/firebase/firebase-js-sdk#automated-setup) 
 for more details.
 
+### Setting up the Firestore emulator
+
+The unit and integration tests require that the Firestore emulator is running
```
Comment (mikelehen) [inline packages/firestore/CONTRIBUTING.md]: nit: I _think_ all the unit tests shouldn't connect to the backend at all.

**Pull request merged**